Sources/core/objects/Interface.cpp

```diff
@@ -86,6 +92,19 @@ Slot<Item>& HumanInterface::GetActiveHand()
         return l_hand_;
 }
 
+void HumanInterface::UpdatePulling(bool isPulling)
+{
+    pulling_ = isPulling;
```

Comment:
Also, pull button has states. When you pull something the image on the button is changed, so I think here is perfect place to implement this.